### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by eroiyo**
# Description

The app retrieves shows information from the external API
likes and comments are then being posted to an involvement API
lastly, likes and comments are being fetched back to the client from the involvement API

Explaining Video
https://drive.google.com/file/d/1tDmLnECZqp1AxTJDGRUd_38L5tzc50Zl/view?usp=sharing

Technologies #

*NPM 
*Webpack
*JavaScript and, JavaScript API's

Features#

- [ ] 2 movies sections with different genres
- [ ] Pop-up messages than display more information about the show or movie
- [ ] Navbar and footer
- [ ] Functional likes and comment system

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

### CODE REVIEW

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @erezfree29 @eroiyo 👋, 

Good job so far! 💪 
* Nice Video presentation ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* I suggest you remove the `console.log` statement from your codebase as it could expose sensitive information
* Add test for all counter functions(ie home page, likes and comments counter)
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/scripts/fetch.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+};
+
+export const Appcreation = () => {
+  const request = new XMLHttpRequest();
+  const requestURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+  request.open('POST', requestURL, true);
+  request.setRequestHeader('Content-type', 'application/x-www-form-urlencoded');
+  request.onreadystatechange = () => {
+    if (request.readyState === 4 && request.status === 201) {
+      // console.log(request.responseText);
```
Comment (Dipeshtwis) [inline src/scripts/fetch.js:24]: [Optional]
-  I would recommend removing the commented code as it makes the codebase cleaner. As it is the only place I'm seeing the commented code, I'm approving but make sure to remove the commented code in general as well when moving code to production in the future as well.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @eroiyo and @erezfree29, 

### Status: Approved ✔️

:+1:  Professional repo, with good commit message, PR title and description, keep up the good work :muscle:
:+1: You have implemented all the previous reviewer's suggestions. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**



**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/scripts/fetch.js (hunk 1 above)
Comment (eroiyo) [reply src/scripts/fetch.js:24]: Sure, we will do!

**Pull request merged**